core/templates/layout.guest.php

```diff
@@ -35,7 +35,7 @@
 				<?php if ($_['bodyid'] === 'body-login'): ?>
 					<header role="banner">
 						<div id="header">
-							<div class="logo"></div>
+							<img class="logo" alt="<?php p($l->t('%s logo', [$theme->getName()])); ?>" src="<?= $_['logoUrl'].$theme->getLogo(); ?>">
```

Comment:
``occ config:system:set logo_url --value="https://nextcloud.com/blog/press_releases/nextcloud-hub-3-release-ai-photos/"`` breaks this. The logo URL can come from config or themeing or both.

```suggestion
							<img class="logo" alt="<?php p($l->t('%s logo', [$theme->getName()])); ?>" src="<?= $_['logoUrl'] ?? $theme->getLogo(); ?>">
```